packages/@okta/vuepress-site/.vuepress/scripts/build-guides-info.js

```diff
@@ -21,26 +21,35 @@ const getFrameworksFor = path => {
 
 const guideInfo = {};
 
+// Load frontmatter yaml to JS from root file index page
```

Comment:
+1 on the comments

Reply:
I had to write all of them while I was originally trying to understand the code so I figured I should just leave them 😄

Reply:
+1, also I was thinking about refactoring (or splitting) `getGuides` method. Right now it looks a bit complicated